src/treeval/scripts/graphing.py

```diff
@@ -89,9 +89,10 @@ def graph_peak_vs_clade(data_df: pl.DataFrame):
 
     unique_subwf = set(data2.get_column("major_subworkflow"))
 
+    fig, axes = plt.subplots(1, 2, figsize=(24, 12))
```

Comment:
```suggestion

```

Reply:
This results in an empty plot, it should be re-added to line 94.

Reply:
Thank you for your feedbacks. I have adjusted the code as requested.